src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+          class="input"
+          name="surname"
+          type="text"
+          id="surname"
+          autocomplete="off"
+          required
+        >
+        <br>
```

Comment:
Why do you need to add `<br>` after every input? It's a strange solution, we shouldn't use additional html elements if we are able to do same thing via CSS

Let's just wrap inputs in label and make blocks from label